src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task{
+    protected String by;
+
+    public Deadline (String name, boolean isCompleted, String by) {
```

Comment:
Can standardize across files to have no space between method name and open parenthesis.

Reply:
thank you